recycleIT-frontend/src/Components/Main/Main.tsx

```diff
@@ -110,6 +111,7 @@ const Main = () => {
                     </Box>
                 </TabContext>
             </Box>
+            <Footer/>
```

Comment:
Footer should only be on the main page? - if not and it should be applied to all pages please make sure to move it to App layout, similar to header

Reply:
Yes) We need Footer only on the main page.